src/components/zoom-to-marker.js

```diff
@@ -0,0 +1,14 @@
+import { useMap } from "react-leaflet"
+
+export default function PanToMarker(position) {
+    const map = useMap()
+    const currentLocation = map.getCenter()
+    const location = position.position ? position.position : [51.505, -0.09]
+
+        map.flyTo( location,10 );
```

Comment:
I prefer lines ending in `;`, but other lines are not - so we are consistent maybe we need an issue for a linter?